src/NovelRT.DotNet/Interop/Audio/NovelRT.cs

```diff
@@ -9,54 +9,54 @@ public static unsafe partial class NovelRT
-        public static extern NrtResult Nrt_AudioService_stopMusic([NativeTypeName("NrtAudioServiceHandle")] IntPtr service);
+        public static extern NrtResult Nrt_AudioService_StopMusic([NativeTypeName("NrtAudioServiceHandle")] IntPtr service);
 
         [DllImport("NovelRT.Interop", ExactSpelling = true)]
-        public static extern NrtResult Nrt_AudioService_setMusicVolume([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, float value);
+        public static extern NrtResult Nrt_AudioService_SetMusicVolume([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, float value);
 
         [DllImport("NovelRT.Interop", ExactSpelling = true)]
-        public static extern NrtResult Nrt_AudioService_checkSources([NativeTypeName("NrtAudioServiceHandle")] IntPtr service);
+        public static extern NrtResult Nrt_AudioService_CheckSources([NativeTypeName("NrtAudioServiceHandle")] IntPtr service);
 
         [DllImport("NovelRT.Interop", ExactSpelling = true)]
-        public static extern NrtResult Nrt_AudioService_loadSound([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, [NativeTypeName("char *")] sbyte* input, [NativeTypeName("unsigned int *")] uint* output);
+        public static extern NrtResult Nrt_AudioService_LoadSound([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, [NativeTypeName("char *")] sbyte* input, [NativeTypeName("unsigned int *")] uint* output);
 
         [DllImport("NovelRT.Interop", ExactSpelling = true)]
-        public static extern NrtResult Nrt_AudioService_unload([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, [NativeTypeName("unsigned int")] uint handle);
+        public static extern NrtResult Nrt_AudioService_Unload([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, [NativeTypeName("unsigned int")] uint handle);
 
         [DllImport("NovelRT.Interop", ExactSpelling = true)]
-        public static extern NrtResult Nrt_AudioService_playSound([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, [NativeTypeName("unsigned int")] uint handle, int loops);
+        public static extern NrtResult Nrt_AudioService_PlaySound([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, [NativeTypeName("unsigned int")] uint handle, int loops);
 
         [DllImport("NovelRT.Interop", ExactSpelling = true)]
-        public static extern NrtResult Nrt_AudioService_stopSound([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, [NativeTypeName("unsigned int")] uint handle);
+        public static extern NrtResult Nrt_AudioService_StopSound([NativeTypeName("NrtAudioServiceHandle")] IntPtr service, [NativeTypeName("unsigned int")] uint handle);
 
         [DllImport("NovelRT.Interop", ExactSpelling = true)]
-        public static extern NrtResult Nrt_AudioService_tearDown([NativeTypeName("NrtAudioServiceHandle")] IntPtr service);
+        public static extern NrtResult Nrt_AudioService_TearDown([NativeTypeName("NrtAudioServiceHandle")] IntPtr service);
     }
 }
```

Comment:
Oh wow, nice! I forgot we needed this! Good catch :D

Reply:
Thank CLion lol